pkg/apis/pipeline/v1beta1/pipeline_validation_test.go

```diff
@@ -1160,14 +1160,32 @@ func TestValidatePipelineResults_Failure(t *testing.T) {
 		results       []PipelineResult
 		expectedError apis.FieldError
 	}{{
-		desc: "invalid pipeline result reference",
+		desc: "invalid pipeline task result reference in parse expression",
```

Comment:
how is this test case different from the one following it?

Reply:
it is not different, unsure why I put it there. has been deleted.